### ISSUE TITLE
Upload is not triggered

### ISSUE THREAD

**Issue by SystemZ** [jvsteiner/s3-image-uploader#18]
Hi, I'm trying to move my attachments from obsidian vault to self-hosted S3 compatible object.
I use Obsidian 1.5.8 on macOS 14.3.1 with s3-image-uploader v0.2.8.

I installed and configured plugin, checked dev console, but pasting or drag and dropping an image or PDF doesn't work.
There is no network activity or errors/warnings in console. 
File is just handled as usual, with local copying to attachments folder.
Is seems like some event is not being handled when I'm adding file to note.

I tried other plugins similar to this and file upload is triggered correctly.
Maybe it's related somehow to newer version of Obsidian?

**Comment by jvsteiner** [comment]
did you enable it?
works on my machine - I can't reproduce based on your description.

**Comment by SystemZ** [comment]
Hi, yes it works now! 😎

First error I made was assuming that network request will show similar to other plugins in network tab of dev tools.
Second, I misconfigured my selfhosted S3 compatible storage, it didn't verify request because of other domain name.
Third, I skipped region part of config in plugin.
Fourth, maybe there was some kind of random activation of plugin problem and it worked after few restarts of obsidian. More probable is a mix of my 3 previous errors.

Thanks for awesome plugin, it greatly helped me maintaining size of vault to acceptable levels 🥳

Related to issue title, can I use this plugin with mobile version?
Can't find method how to handle it with iOS share to Obsidian. 
My main use case would be adding photo I made to obsidian with storage on S3 quickly.
